textile/features.textile

```diff
@@ -654,6 +655,7 @@ h3(#realtime-channel). RealtimeChannel
 * @(RTL6)@ @RealtimeChannel#publish@ function:
 ** @(RTL6a)@ Messages are encoded in the same way as the @RestChannel#publish@ method, and "RSL1g":#RSL1g (size limit) applies similarly
 *** @(RTL6a1)@ "RSL1k":#RSL1k (@idempotentRestPublishing@ option), "RSL1j1":#RSL1j1 (idempotent publishing test), and "RSL1l":#RSL1l (@publish(Message, params)@ form) do not apply to realtime publishes
+*** @(RTL6a2)@ Just like "RSL1j1":#RSL1j1 (publishing via REST), @Message@ objects can have the "TM2j2":#TM2j2 (message action field) set to @MESSAGE_CREATE@, if unset, this will be set by realtime to @MESSAGE_CREATE@.
```

Comment:
```suggestion
*** @(RTL6a2)@ Similar to "RSL1j1":#RSL1j1 (publishing via REST), @Message@ objects can have the "TM2j2":#TM2j2 (message action field) set to @MESSAGE_CREATE@, if unset, this will be set by realtime to @MESSAGE_CREATE@.
```

Sounds a bit more formal :)

Reply:
Removed